backend/src/util.rs

```diff
@@ -183,15 +183,13 @@ pub struct Price {
 /// This struct contains all environmental information. co2 in gramms, water in litres
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct EnvironmentInfo {
-    /// The number of stars the food has on average. Out of `max_rating`
-    pub average_rating: u32,
     /// The number of stars the food has for co2 emmissions. Out of `max_rating`
     pub co2_rating: u32,
-    /// The amount of co2 emitted by the production of the food
+    /// The amount of co2 emitted by the production of the food. In gramms
```

Comment:
```suggestion
    /// The amount of CO2 emitted by the production of the food. In grams.
```
same for milliliters. Important information 👍 .